spec/models/reservation_spec.rb

```diff
@@ -494,6 +494,39 @@
     end
   end
 
+  context 'when manually over-extending reservation' do
+    before(:each) do
+      @em = FactoryGirl.create(:equipment_model)
+      @ei = FactoryGirl.create(:equipment_item, equipment_model: @em)
+
+      FactoryGirl.create(:valid_reservation)
+      reservation.equipment_item  = @ei
+      reservation.equipment_model = @em
+      reservation.start_date      = Time.zone.today + 1.day
+      reservation.due_date        = Time.zone.today + 2.day
+
+      @r2 = FactoryGirl.build(:valid_reservation)
```

Comment:
See above, you can consolidate this into a single call (`FactoryGirl.build(:valid_reservation, equipment_model: @em, ...)`)